src/main/webapp/app/exercises/text/assess/text-assessment-dashboard/text-assessment-dashboard.component.ts

```diff
@@ -119,4 +128,12 @@ export class TextAssessmentDashboardComponent implements OnInit {
             });
         }
     }
+
+    /**
+     * Gets the duration of the submission in minutes.
+     * @param exercise The exercise object.
+     */
+    durationForSubmission(submission: Submission) {
+        return this.momentDiff.transform(submission.submissionDate!, submission.participation.initializationDate!, 'minutes');
```

Comment:
what happens if the submission date is null/undefined?

Reply:
Addressed via: 55d5b75